packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -534,16 +534,30 @@ class TextSelectionOverlay {
         break;
     }
 
-    _handleSelectionHandleChanged(newSelection, isEnd: false);
+    _handleSelectionHandleChanged(newSelection, handleSide: TextSelectionDraggedHandleSide.baseHandle);
   }
 
-  void _handleSelectionHandleChanged(TextSelection newSelection, {required bool isEnd}) {
-    final TextPosition textPosition = isEnd ? newSelection.extent : newSelection.base;
+  void _handleSelectionHandleChanged(TextSelection newSelection, {required TextSelectionDraggedHandleSide handleSide}) {
+    final TextPosition positionToView;
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.iOS:
+      case TargetPlatform.macOS:
+        // For iOS and macOS, extent always means the dragged handle by this change https://github.com/flutter/flutter/pull/105069
```

Comment:
Instead of "by this change" perhaps we do something like:
// On iOS and macOS, extent is always the dragged handle
// (PR LINK IN PARENS).